Instanciador iniciado correctamente
-------------------- - 
Se realiza login de manera correcta
Se accede correctamente a Gestión de Cliente
Se accede correctamente a la sección Mis Ofertas lyntia
Nueva Oferta creada correctamente
Se han introducido correctamente los campos de la Oferta: CRM-COF0005, CLIENTE INTEGRACION, Nuevo servicio, # BizQA.
La Oferta se guarda correctamente.
***Se cumplen las condiciones de guardado de oferta correctamente
Se accede correctamente a la pestaña Fechas de la Oferta.
***Las condiciones de fechas informadas correctamente han sido OK
Se elimina la Oferta correctamente.
Se pulsa correctamente Eliminar.
Se accede correctamente a la sección Mis Ofertas lyntia
Nueva Oferta creada correctamente
Se han introducido correctamente los campos de la Oferta: CRM-COF0005, CLIENTE INTEGRACION, Nuevo servicio, # BizQA.
La Oferta se guarda y cierra correctamente.
Se busca la Oferta por parámetro: CRM-COF00052802230823.
***Se cumple la condición de Oferta guardada Correctamente.
Se abre la Oferta CRM-COF00052802230823 se abre correctamente.
***No se cumplen las condiciones para guardar la oferta
-------------------- - 
-------------------- - 
Excepción: OpenQA.Selenium.WebDriverTimeoutException: Timed out after 45 seconds
 ---> OpenQA.Selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//input[@aria-label='Nombre oferta']"}
  (Session info: chrome=110.0.5481.178)
   at OpenQA.Selenium.WebDriver.UnpackAndThrowOnError(Response errorResponse, String commandToExecute)
   at OpenQA.Selenium.WebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)
   at OpenQA.Selenium.WebDriver.FindElement(String mechanism, String value)
   at OpenQA.Selenium.By.<.ctor>b__11_0(ISearchContext context)
   at OpenQA.Selenium.By.FindElement(ISearchContext context)
   at OpenQA.Selenium.WebDriver.FindElement(By by)
   at SeleniumExtras.WaitHelpers.ExpectedConditions.<>c__DisplayClass19_0.<ElementToBeClickable>b__0(IWebDriver driver)
   at OpenQA.Selenium.Support.UI.DefaultWait`1.Until[TResult](Func`2 condition, CancellationToken token)
   --- End of inner exception stack trace ---
   at OpenQA.Selenium.Support.UI.DefaultWait`1.ThrowTimeoutException(String exceptionMessage, Exception lastException)
   at OpenQA.Selenium.Support.UI.DefaultWait`1.Until[TResult](Func`2 condition, CancellationToken token)
   at OpenQA.Selenium.Support.UI.DefaultWait`1.Until[TResult](Func`2 condition)
   at Lyntia.TestSet.Actions.CommonActions.waitElement(By locator) in C:\Users\arnau.suiles.ext\source\repos\Lyntia-automation\Lyntia\TestSet\Actions\CommonActions.cs:line 262
   at Lyntia.TestSet.Actions.CommonActions.getElement(By locator) in C:\Users\arnau.suiles.ext\source\repos\Lyntia-automation\Lyntia\TestSet\Actions\CommonActions.cs:line 236
   at Lyntia.TestSet.Conditions.OfertaConditions.validar_OfertaGuardadaCorrectamente(Dictionary`2 datosTestCase) in C:\Users\arnau.suiles.ext\source\repos\Lyntia-automation\Lyntia\TestSet\Conditions\OfertaConditions.cs:line 117
